snap/snapcraft.yaml

```
@@ -57,33 +57,20 @@ parts:
 
   zap:
     plugin: nil
-    source: https://github.com/project-chip/zap.git
-    source-depth: 1
-    source-tag: v2023.05.04
     build-environment:
-      - NODE_VERSION: v18.16.1
+      - ZAP_VERSION: v2023.09.28
     build-packages:
       - build-essential 
-      - libpango1.0-dev 
-      - libjpeg-dev 
-      - libgif-dev 
-      - librsvg2-dev
       - wget
+      - unzip
     override-build: |
       if [[ $SNAP_ARCH == "arm64" ]]; then
-        # Install node+npm
-        NODE=node-$NODE_VERSION-linux-$SNAP_ARCH
-        wget https://nodejs.org/dist/$NODE_VERSION/$NODE.tar.xz
-        tar -xJf $NODE.tar.xz
-        export PATH=$PWD/$NODE/bin:$PATH
-
-        # Install zap
-        npm install
+        # Download and unzip the prebuilt ZAP (Zigbee Cluster Library configuration tool and generator) binary for the ARM64 architecture
+        wget https://github.com/project-chip/zap/releases/download/$ZAP_VERSION/zap-linux-$SNAP_ARCH.zip
```

This is a large file, the progress data makes a lot of noise in the logs.
```suggestion
        wget --no-verbose https://github.com/project-chip/zap/releases/download/$ZAP_VERSION/zap-linux-$SNAP_ARCH.zip
```